java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsBaseCommands.java

```diff
@@ -128,4 +132,152 @@ CompletableFuture<Object> fcall(
+     * handled internally. If the script has not already been loaded, it will be loaded automatically
+     * using the <code>SCRIPT LOAD</code> command. After that, it will be invoked using the <code>
+     * EVALSHA</code> command.
+     *
+     * @apiNote When in cluster mode
+     *     <ul>
+     *       <li>all <code>keys</code> must map to the same hash slot.
+     *       <li>if no <code>keys</code> are given, command will be routed to a random primary node.
+     *     </ul>
+     *
+     * @see <a href="https://valkey.io/commands/script-load/">SCRIPT LOAD</a> and <a
+     *     href="https://valkey.io/commands/evalsha/">EVALSHA</a> for details.
+     * @param script The Lua script to execute.
+     * @param options The script option that contains keys and arguments for the script.
+     * @return a value that depends on the script that was executed.
+     * @example
+     *     <pre>{@code
+     * try(Script luaScript = new Script(gs("return { KEYS[1], ARGV[1] }", true))) {
+     *     ScriptOptionsGlideString scriptOptions = ScriptOptionsGlideString.builder().key(gs("foo")).arg(gs("bar")).build();
+     *     Object[] result = (Object[]) client.invokeScript(luaScript, scriptOptions).get();
+     *     assert result[0].equals(gs("foo"));
+     *     assert result[1].equals(gs("bar"));
+     * }
+     * }</pre>
+     */
+    CompletableFuture<Object> invokeScript(Script script, ScriptOptionsGlideString options);
+
+    /**
+     * Check existence of scripts in the script cache by their SHA1 digest.
+     *
+     * @see <a href="https://valkey.io/commands/script-exists">SCRIPT EXISTS</a> for details.
+     * @param sha1s The Lua script to execute.
+     * @return An array of <code>boolean</code> values indicating the existence of each script.
+     * @example
+     *     <pre>{@code
+     * try(Script luaScript = new Script("return { KEYS[1], ARGV[1] }", true)) {
+     *     client.invokeScript(luaScript).get();
+     *     Boolean[] result = client.scriptExists(new String[]{luaScript.getHash()});
+     *     assert result[0].equals(true);
+     * }
+     * }</pre>
+     */
+    CompletableFuture<Boolean[]> scriptExists(String[] sha1s);
+
+    /**
+     * Check existence of scripts in the script cache by their SHA1 digest.
+     *
+     * @see <a href="https://valkey.io/commands/script-exists">SCRIPT EXISTS</a> for details.
+     * @param sha1s The Lua script to execute.
+     * @return An array of <code>boolean</code> values indicating the existence of each script.
+     * @example
+     *     <pre>{@code
+     * try(Script luaScript = new Script(gs("return { KEYS[1], ARGV[1] }", true))) {
+     *     client.invokeScript(luaScript).get();
+     *     Boolean[] result = client.scriptExists(new String[]{luaScript.getHash()});
+     *     assert result[0].equals(true);
+     * }
+     * }</pre>
+     */
+    CompletableFuture<Boolean[]> scriptExists(GlideString[] sha1s);
+
+    /**
+     * Flush the Lua scripts cache.
```

Comment:
```suggestion
     * Flushes the Lua scripts cache.
```
Present tense - update for all docs

Reply:
might have to make a mini pr to fix the same grammar mistakes in python